internal/integrations/v4/runner/runner.go

```diff
@@ -255,6 +280,10 @@ func (r *runner) handleLines(stdout <-chan []byte, extraLabels data.Map, entityR
 		}
 
 		if ok, ver := protocol.IsCommandRequest(line); ok {
+			if r.handleCmdReq == nil {
```

Comment:
this check should go right after `Received run request.` debug entry, otherwise a received request won't be logged

Reply:
@varas the `Warn` log has the payload field, otherwise the payload is printed twice. Here is an example:
```
time="2021-05-20T09:13:22+02:00" level=warning msg="received cmd request payload without a handler" component=integrations.runner.Runner integration_name=cmdreq payload="{  \"command_request_version\": \"1\",  \"commands\": [    {      \"name\": \"cmd-req-name\",      \"command\": \"/Users/gsanchez/go/src/github.com/newrelic/infrastructure-agent/pkg/integrations/v4/fixtures/cmdreq/v4.sh\",      \"args\": [],      \"env\": {        \"FOO\": \"BAR\"      }    }  ]}"

Reply:
the point of `Debug` entries is to follow the trace of such payload along the way, yes, if verbose is enabled it'll be printed twice first one showing always for each request along with the protocol version

but yes, probably I'm adding too much trace context, your approach should be enough, nevermind